doc/examples/viz_advanced.py

```diff
@@ -0,0 +1,187 @@
+"""
+
+res = read_bundles_2_subjects('subj_1', ['t1', 'fa'],
+                              ['af.left', 'cst.right', 'cc_1'])
+
+"""
+We will use 3 bundles, FA and the affine transformation that brings the voxel
+coordinates to world coordinates (RAS 1mm).
+"""
+
+streamlines = res['af.left'] + res['cst.right'] + res['cc_1']
+data = res['fa']
+shape = data.shape
+affine = res['affine']
+
+"""
+With our current design it is easy to decide in which space you want the
+streamlines and slices to appear. The default we have here is to appear in
+world coordinates (RAS 1mm).
+"""
+
+world_coords = True
+
+"""
+If we want to see the objects in native space we need to make sure that all
+objects which are currently in world coordinates are transformed back to
+native space using the inverse of the affine.
+"""
+
+if not world_coords:
+    from dipy.tracking.streamline import transform_streamlines
+    streamlines = transform_streamlines(streamlines, np.linalg.inv(affine))
+
+"""
+Now we create, a ``Renderer`` object and add the streamlines using the ``line``
+function and an image plane using the ``slice`` function.
+"""
+
+ren = window.Renderer()
+stream_actor = actor.line(streamlines)
+
+if not world_coords:
+    image_actor = actor.slicer(data, affine=np.eye(4))
+else:
+    image_actor = actor.slicer(data, affine)
+
+"""
+For fun let's change also the opacity of the slicer
+"""
+
+slicer_opacity = .6
+image_actor.opacity(slicer_opacity)
+
+"""
+Connect the actors with the Renderer.
+"""
+
+ren.add(stream_actor)
+ren.add(image_actor)
+
+"""
+Now we would like to change the position of the ``image_actor`` using a slider.
+The sliders are widgets which require access to different areas of the
+visualization pipeline and therefore we don't recommend using them with
+``show``. The more appropriate way is to use them with the ``ShowManager``
+object which allows accessing the pipeline in different areas. Here is how:
+"""
+
+show_m = window.ShowManager(ren, size=(1200, 900))
+show_m.initialize()
+
+"""
+After we have initialized the ``ShowManager`` we can go ahead and create a
+callback which will be given to the ``slider`` function.
+"""
+
+
+def change_slice(obj, event):
+    z = int(np.round(obj.get_value()))
+    image_actor.display_extent(0, shape[0] - 1,
+                               0, shape[1] - 1, z, z)
+
+slider = widget.slider(show_m.iren, show_m.ren,
+                       callback=change_slice,
+                       min_value=0,
+                       max_value=shape[2] - 1,
+                       value=shape[2] / 2,
+                       label="Move slice",
+                       right_normalized_pos=(.98, 0.6),
+                       size=(120, 0), label_format="%0.lf",
+                       color=(1., 1., 1.),
+                       selected_color=(0.86, 0.33, 1.))
+
+"""
+Then, we can render all the widget and everything else in the screen and
+start the interaction using ``show_m.start()``.
```

Comment:
Maybe add a sentence saying that users will have to uncomment the next couple of lines to see this.